src/app/Archives/AllArchivedRecordingsTable.tsx

```diff
@@ -248,7 +248,9 @@ export const AllArchivedRecordingsTable: React.FunctionComponent<AllArchivedReco
           </ToolbarGroup>
           <ToolbarGroup variant="icon-button-group">
             <ToolbarItem>
-              <Button variant="plain" aria-label="add" onClick={() => setShowUploadModal(true)}><PlusIcon /></Button>
+                <Button key="upload" variant="secondary" onClick={() => () => setShowUploadModal(true)}>
```

Comment:
Clicking on the upload button on the Archives view doesn't work anymore and I think the problem is this line. The `onClick` return type went from `() => void` to `() => ( () => void )`.

Reply:
Oh sorry I think i accidentally typed `() => ` twice.